English_transcripts/school/Epic Quest/sharedAssets/C1_E_EQ_WProbs_num3C.txt

```diff
@@ -0,0 +1 @@
+How many more oranges does mother need so that each child gets one orange?
```

Comment:
How many more oranges does **the** mother need

Reply:
Updated the transcript.